utility/BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+'''A kt_android_library() rule that builds utility module kotlin source files into a library.'''
```

Comment:
We can switch to single line comment `#`

Reply:
Done!